### PR TITLE
destro talent overhaul / minor updates

### PR DESCRIPTION
**Pull request by superderpicus**
@sMteX: added talent hub to clean up the interface. added a couple wording/explanation fixes to ABC module.

that merge is me merging latest wowanalyzer into my fork.
removing combatants broke it so i put it back

### CODE REVIEW

File: src/Parser/Warlock/Destruction/Modules/Features/AlwaysBeCasting.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ class AlwaysBeCasting extends CoreAlwaysBeCasting {
 
     when(deadTimePercentage).isGreaterThan(0.2)
       .addSuggestion((suggest, actual, recommended) => {
-        return suggest(<span>Your downtime can be improved. Try to Always Be Casting (ABC), try to reduce the delay between casting spells. Even if you have to move, try casting something instant - maybe refresh your dots or replenish your mana with <SpellLink id={SPELLS.LIFE_TAP.id} /></span>)
+        return suggest(<span>Your downtime can be improved. Try to Always Be Casting (ABC), try to reduce the delay between casting spells. Even if you have to move, try casting something instant - maybe save dimensional rift charges, conflagrate, or replenish your mana with <SpellLink id={SPELLS.LIFE_TAP.id} />. You should make good use of demonic circle if you can.</span>)
```
Comment (sMteX) [inline src/Parser/Warlock/Destruction/Modules/Features/AlwaysBeCasting.js]: I would suggest a bit of more rewording, sounds a bit too clunky to me. How about something like "When you know you'll be moving, try to save Dimensional Rift or Conflagrate charges or replenish your mana with Life Tap. Make good use of Demonic Circle when you can." ?

Also, please add SpellLink to Dimensional Rift, Conflagrate, Life Tap (that one's already there) and Demonic Circle.

Comment (superderpicus) [reply src/Parser/Warlock/Destruction/Modules/Features/AlwaysBeCasting.js]: ok

Comment (superderpicus) [reply src/Parser/Warlock/Destruction/Modules/Features/AlwaysBeCasting.js]: here is the preview https://i.imgur.com/XtkYlc6.jpg, also i added the wording changes/spell links.

**Comment by superderpicus** [comment]
@sMteX https://i.imgur.com/XtkYlc6.jpg https://i.imgur.com/UwzQ72L.png examples

**Pull request merged**